Tests/features/CuckooGeneratorTests.feature

```diff
@@ -0,0 +1,5 @@
+Feature: CuckooGeneratorTests
+	Scenario: Generating test mocks
+		Given I run `xcodebuild -project '../../../CuckooGenerator.xcodeproj' -scheme 'CuckooGenerator' -configuration 'Release' CONFIGURATION_BUILD_DIR=Tests/tmp/aruba clean build`
+		When I run `tmp/aruba/cuckoo.app/Contents/MacOS/cuckoo generate --no-header --output actual.swift ../../res/source/TestedClass.swift ../../res/source/TestedProtocol.swift`
+		Then the file "../../res/expected.swift" should be equal to file "actual.swift"
```

Comment:
The paths in the test are too verbose as well, we need to make it more easy and pleasant to write those test. There will have to be a lot of them.